src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -110,23 +125,38 @@ class ActivityFeed extends React.Component<Props, State> {
             currentUser: prevCurrentUser,
             feedItems: prevFeedItems,
         } = prevProps;
-        const { feedItems: currFeedItems, activeFeedEntryId } = this.props;
-        const { isInputOpen: prevIsInputOpen } = prevState;
-        const { isInputOpen: currIsInputOpen } = this.state;
+        const { feedItems: currFeedItems, activeFeedEntryId, onShowReplies = noop } = this.props;
+        const { activeEntry, isInputOpen: prevIsInputOpen } = prevState;
+        const { activeEntry: prevActiveEntry, isInputOpen: currIsInputOpen } = this.state;
```

Comment:
Is this inverted? It looks like this should be `currActiveEntry` and the line above should be `prevActiveEntry`.